### PR TITLE
Https npm-run-all for local-ssl-proxy

### LINKED ISSUE #202: Single command to run dev and start https
Using https://www.npmjs.com/package/npm-run-all to start the next dev server and also the local-ssl-proxy for https

### PR DESCRIPTION
**Pull request by ankushdharkar**
Now, we just have to run `npm run dev` and the https server spins up. No more dual-terminal execution
Closes #202

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm-run-all --parallel dev:*",
+    "dev:next": "next dev -p 3443",
+    "dev:https-link": "local-ssl-proxy --source 443 --target 3443",
```
Comment (swarajpure) [inline package.json:8]: Won't we be forcing everyone to use https in this case? Like if someone isn't working on API related stuff, they still will have to setup the https, is it so? I haven't used it, so I'm not sure. Please clarify.

Newcomers might have to undergo more hurdles setting something up which they might not even what about 😅

File: package.json [hunk 2]
```diff
@@ -19,6 +21,7 @@
     "chart.js": "2.9.4",
     "eslint-plugin-jest": "^24.1.3",
     "framer-motion": "^2.9.4",
+    "local-ssl-proxy": "^1.3.0",
```
Comment (swarajpure) [inline package.json:24]: Shouldn't this be a dev dependency? 🤔

**Review by swarajpure — COMMENTED** [review]
Won't we be forcing everyone to use https in this case? Like if someone isn't working on API related task, they still will have to setup the https, is it so? I haven't used it, so I'm not sure. Please clarify.

Newcomers might have to undergo more hurdles setting something up which they might not even what about 😅

File: package.json (hunk 1 above)
Comment (ankushdharkar) [reply package.json:8]: The https setup is automatic
You can still use the site on localhost:3443, but I would discourage that and encourage devs to start getting on https:// for development too

Comment (swarajpure) [reply package.json:8]: Does that mean localhost:3443 would be on http by default, is it so? Or it'll be https://localhost:3443?

Comment (ankushdharkar) [reply package.json:8]: http://localhost:3443
https://localhost
https://localhost:443
https://dev.realdevsquad.com

File: package.json (hunk 2 above)
Comment (ankushdharkar) [reply package.json:24]: Yes! It should be. Good catch!

Comment (ankushdharkar) [reply package.json]: Fixed it.

File: package.json (hunk 1 above)
Comment (swarajpure) [reply package.json:8]: Cool

**Review by swarajpure — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**